BMM.Api/Implementation/Models/DiscoverSectionHeader.cs

```diff
@@ -22,11 +22,14 @@ public DiscoverSectionHeader()
         public string TranslationId { get; set; }
 
         public bool UseCoverCarousel { get; set; }
-
+        
         [JsonIgnore]
         public bool HasLink => string.IsNullOrEmpty(Link);
 
         [JsonIgnore]
         public bool IsSeparatorVisible { get; set; } = true;
+
+        [JsonIgnore]
+        public string Origin { get; set; }
```

Comment:
shouldn't those properties be moved into the PO?

Reply:
done